### PR TITLE
markdig fixes - Part 3

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/System.Threading.Tasks/TaskFactory`1.xml [hunk 1]
```diff
@@ -596,7 +596,7 @@
       <Docs>
         <param name="tasks">The array of tasks from which to continue.</param>
         <param name="continuationFunction">The function delegate to execute asynchronously when all tasks in the <paramref name="tasks" /> array have completed.</param>
-        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The NotOn* or OnlyOn* values are not valid.</param>
+        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The <see langword="NotOn*" /> or <see langword="OnlyOn*" /> values are not valid.</param>
```
Comment (rpetrusha) [inline xml/System.Threading.Tasks/TaskFactory`1.xml]:
```suggestion
        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The <see langword="NotOn*" /> and <see langword="OnlyOn*" /> values are not valid.</param>
```

File: xml/System.Threading.Tasks/TaskFactory`1.xml [hunk 2]
```diff
@@ -596,7 +596,7 @@
       <Docs>
         <param name="tasks">The array of tasks from which to continue.</param>
         <param name="continuationFunction">The function delegate to execute asynchronously when all tasks in the <paramref name="tasks" /> array have completed.</param>
-        <param name="continuationOptions">One of the enumeration values that controls the behavior of the created continuation task. The NotOn* or OnlyOn* values are not valid.</param>
```
Comment (rpetrusha) [inline xml/System.Threading.Tasks/TaskFactory`1.xml]: The strings `NotOn* or OnlyOn*` and `NotOn* and OnlyOn*` are found in a number of other overloads with a TaskContinuationOptions parameter. Do you want to change those as well?

Comment (mairaw) [reply xml/System.Threading.Tasks/TaskFactory`1.xml]: You mean in other types @rpetrusha?

Comment (rpetrusha) [reply xml/System.Threading.Tasks/TaskFactory`1.xml]: No. At least I thought that there were additional occurrences in this type.

Comment (mairaw) [reply xml/System.Threading.Tasks/TaskFactory`1.xml]: I just fixed those.

**Pull request merged**